src/aria/widgets/action/Button.js

```diff
@@ -149,29 +149,24 @@ module.exports = Aria.classDefinition({
             if (!skipChangeState) {
                 // force widget - DOM mapping
                 this.getDom();
-                if (this._simpleHTML) {
-                    if (state == "disabled") {
-                        this._focusElt.setAttribute("disabled", "disabled");
-                    } else {
-                        this._focusElt.removeAttribute("disabled");
+                this._frame.changeState(this._state);
```

Comment:
There is no frame when `this.simpleHtml` is true, so this line will raise an exception.
It is strange that unit tests did not catch this problem!